keyboards/ft/mars65/readme.md

```diff
@@ -0,0 +1,21 @@
+# Mars 6.5
+
+65% Keyboard with in switch backlight and RGB Underglow. 
+
+![Mars 6.5 image](https://i.imgur.com/cg5qADrl.jpg)
```

Comment:
Imgur link no longer works, when you replace it you can append one of the following sizes to make it more appropriately sized:

```
s =   90×  90 = Small Square (as seen in the example above)
b =  160× 160 = Big Square 
t =  160× 160 = Small Thumbnail 
m =  320× 320 = Medium Thumbnail 
l =  640× 640 = Large Thumbnail
h = 1024×1024 = Huge Thumbnail
```

I.e. for the original link `https://i.imgur.com/cg5qADrlh.jpg`

Reply:
> `https://i.imgur.com/cg5qADrlh.jpg`
Doesn't seem to work, checked with curl too to be 100% sure that it wasn't a cache error and it returns a 302, I replaced it with `https://i.imgur.com/cg5qADrh.jpg` instead that also double checking with curl returns a 200.

Reply:
No worries, I was maybe being overzealous as your image actually is appropriately sized already, sorry!